.github/workflows/gitian.yml

```diff
@@ -0,0 +1,44 @@
+name: ci/gh-actions/gitian
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-gitian:
+    runs-on: ubuntu-20.04
```

Comment:
https://github.com/monero-project/monero/blob/master/contrib/gitian/README.md says
> Gitian host OS should be Ubuntu 18.04 "Bionic Beaver". If you are on a mac or windows for example, you can run it in a VM but will be slower.

I know that Github is going to deprecate 18.04 soon, but changing OS to 20.04 will probably create different binaries.

Reply:
I did confirm that the hashes match.

Reply:
The we can probably change the README as well

Reply:
Gitian creates its own container, built on Ubuntu 18.04 regardless. Thus the binaries will always be the same. The difference in host OS can affect the format of the .assert files, which are generated by host tools, but that won't matter if nobody is expecting to sign these builds.